scripts/run_performance_job.py

```diff
@@ -541,6 +541,17 @@ def run_performance_job(args: RunPerformanceJobArgs):
         configurations["CompilationMode"] = "JIT"
         configurations["RuntimeType"] = str(runtime_type)
 
+    if args.run_kind == "maui_scenarios_android":
+        if args.runtime_flavor == "mono":
+            runtime_type = "Mono"
+            configurations["CompilationMode"] = "ProfiledAOT"
```

Comment:
We should make sure other similar android app testing matches this for Mono.

Reply:
I added a comment explaining which scenario is which. We run different configuration for dotnet/runtime android sample than the default configuration used in .NET Android / .NET MAUI Android samples.